arch/arm/core/aarch32/cortex_m/timing.c

```diff
@@ -26,7 +26,10 @@
  */
 static inline uint64_t z_arm_dwt_freq_get(void)
 {
-#if defined(CONFIG_SOC_FAMILY_NRF)
+#if defined(CONFIG_SOC_FAMILY_NRF) || \
+	defined(CONFIG_SOC_FAMILY_IMX) || \
+	defined(CONFIG_SOC_FAMILY_KINETIS) || \
+	defined(CONFIG_SOC_FAMILY_LPC)
```

Comment:
Should we really be changing IMX and KINETIS?

Reply:
IMX was a mistake. I have changed this to RT family. I have added NXP families in anticipation future power management changes.

Reply:
I have updated this code to make this change for RT600 only. We can add changes for LPC and other RT devices later whenever it is required.